config/src/config/test_data/public_full_node.yaml

```diff
@@ -1,18 +1,33 @@
 base:
+    # Update this value if you would like to connect to a different network
     chain_id: "TESTING"
+    # Update this value to the location you want Libra to store its database
     data_dir: "/opt/libra/data"
     role: "full_node"
     waypoint:
+        # Update this value to that which Libra publicly provides
```

Comment:
this is great! suggestion: remove "update this value" since this is a configuration and any value can in theory be updated :p I think replacing with an explanation of what the field is might be better? Unless you're doing this specifically for the drills/ceremony